.github/workflows/cd-analytics-infra.yml

```
@@ -0,0 +1,61 @@
+name: Deploy Analytics Infra
```

```suggestion
name: Deploy Analytics Infra
run-name: Deploy ${{ github.ref_name }} to Analytics ${{ (github.event_name == 'release' && 'prod') || ['dev', 'staging'] }}
```

Add `run-name` as well, from here: https://github.com/HHS/simpler-grants-gov/blob/main/.github/workflows/cd-api.yml#L4

Also add it to the other files!